website/static/css/custom.css

```diff
@@ -0,0 +1,14 @@
+/* Custom styles */
+
+.warning   {
+    background: #cc2020;
+    padding: 4pt;
+    border-radius: 6pt;
+    color: #fff;
+}
+
+div.warning > h3, 
+div.warning > h4   {

```

Comment:
```suggestion
.warning > h3, 
.warning > h4   {

```

Reply:
@chrisglein  that no worky:
![image](https://user-images.githubusercontent.com/22989529/92951808-d565dc80-f413-11ea-82ef-bb5f54554f76.png)
![image](https://user-images.githubusercontent.com/22989529/92951828-e0b90800-f413-11ea-8812-98f7ab2b5f81.png)

Reply:
> @chrisglein that no worky:

Ugh, the new style isn't winning because it's earlier in the file and the rules have the same specificity. CSS rules don't just go in order in the file, they're also weighed by specificity. So `.mainContainer h3` is the same level as `.warning > h3` . By making `div.warning > h3` you're increasing the specificity enough to override the file order. Alternate fixes would be either:
- Move the `.warning` CSS rules lower in the file
- Update the rule `.mainContainer .warning h3` so that it is more specific in a way that better matches the overriden rule

Reply:
@chrisglein we don't control the order of inclusion as that's done by docusaurus. 
If/we move to docusaurus v2, a lot of this can go away since we can just embed react components instead of dealing with css styling

Reply:
filed #269, I've been talking to the docusaurus dev about migrating, we need some features from v1 that were not ported to v2 yet before we can move.